src/Kunstmaan/AdminBundle/Form/UserType.php

```diff
@@ -106,8 +106,10 @@ public function getName()
      */
     public function setDefaultOptions(OptionsResolverInterface $resolver)
     {
-        $resolver->setDefaults(array('password_required' => false));
+        $resolver->setDefaults(
+            array('password_required' => false,
+                  'data_class' => 'Kunstmaan\AdminBundle\Entity\User',
```

Comment:
The data class should not be hardcoded here, as users can override it with their own implementation, and you'll need that class name if they do.